Acknowledgements.txt

```diff
@@ -4410,3 +4410,37 @@ products or services of Licensee, or any third party.
 8. By copying, installing or otherwise using Python, Licensee
 agrees to be bound by the terms and conditions of this License
 Agreement.
+
+==============================================================================
+str2bool
```

Comment:
Let's hope it won't end up similarly to leftpad.

Reply:
We have own copy of that code in DALI - docs/examples/use_cases/paddle/resnet50/utils/config.py